### ISSUE TITLE
[Feature] Please add more rescheduling flexibility

### ISSUE THREAD

**Issue by AlexeyMalafeev** [ankidroid/Anki-Android#8952]
Tldr: 

I'd like AnkiDroid to have a simple and user-friendly way of cutting a card's interval (e.g. by half) without penalizing the card's ease. 

Problem:

AFAIK, right now ease always gets reduced by 20 every time you choose Hard, to the limit of 130. This leads to the so-called 'low ease hell'. To avoid this, one could manually reschedule (in particular, reduce the interval of) cards, which resets the ease to 250. Right now, I use 'reschedule for review in X days' for this exact purpose (or sometimes also to rescue my old 130-ease cards from hell), on a daily basis. I access this option via the menu, which takes two taps: Reschedule -> reschedule for review in X days. 

Possible solutions:

A) Please consider adding 'reschedule for review in X days' to actions assignable to gestures. This will save taps. 

B) Please consider adding an (optional) button 'Reschedule for review in half the current interval' (while resetting the card's ease to 250, as usual). 

C) Alternatively, please consider adding the option of not reducing the card's ease when Hard is tapped. Actually, this might be the best (most straightforward) solution for my use case.

**Comment by mikehardy** [comment]
Hi there - I think all of this discussion belongs upstream - it all seems reasonable but we follow upstream features and scheduling as exactly as possible - forums.ankiweb.net/

If they can be convinced, we will implement it following them as quickly as possible

Were we to deviate, the cross-platform nature of the Anki system would be eroded and that's a strong anti-goal

**Comment by AlexeyMalafeev** [comment]
@mikehardy I got your point about upstream features, but what about my suggestion A? (Please consider adding 'reschedule for review in X days' to actions assignable to gestures. This will save taps.)

I think this is purely AnkiDroid front-end and does not depend on Anki features. I mean, 'reschedule for review in X days' is already available on AnkiDroid, I am just asking for it to be accessible more quickly. One tap (gesture) is better than two...

**Comment by mikehardy** [comment]
We attempt to match front-end features as well

**Comment by AlexeyMalafeev** [comment]
Ok, got it.
